apps/site/pages/docs/customizing/mixins-breakpoints.mdx

```diff
@@ -0,0 +1,246 @@
+      @if $position == "top" {
+        padding-top: $padding;
+        border-top: var(--fs-border-width) solid var(--fs-border-color-light);
+
+        @include media(">=notebook") { padding-top: $padding; }
+      }
+
+      @if $position == "bottom" {
+        padding-bottom: $padding;
+        border-bottom: var(--fs-border-width) solid var(--fs-border-color-light);
+
+        @include media(">=notebook") { padding-bottom: $padding; }
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+    ```scss copy
+    @mixin input-focus-ring($outline: #{var(--fs-color-focus-ring)}, $border: #{var(--fs-border-color-active)}) {
+      @media not all and (min-resolution: .001dpcm) { // Target only Safari browsers
+        @supports (-webkit-appearance:none) {         // Use `focus` instead of `focus-visible`
+          &:hover:focus,                              // due to Safari's lack of support
+          &:focus {
+            border-color: $border;
+            outline: none;
+            box-shadow:
+              0 0 0 1px var(--fs-color-body-bkg),
+              0 0 0 var(--fs-border-width-thickest) $outline,
+              inset 0 0 0 var(--fs-border-width) $border;
+          }
+        }
+      }
+
+      &:focus-visible,
+      &:hover:focus-visible {
+        border-color: $border;
+        outline: none;
+        box-shadow:
+          0 0 0 1px var(--fs-color-body-bkg),
+          0 0 0 var(--fs-border-width-thickest) $outline,
+          inset 0 0 0 var(--fs-border-width) $border;
+      }
+    }
+
+    @mixin focus-ring {
+      outline: none;
+      box-shadow: 0 0 0 1px var(--fs-color-body-bkg), 0 0 0 var(--fs-border-width-thickest) var(--fs-color-focus-ring);
+    }
+
+    @mixin focus-ring-visible {
+      @media not all and (min-resolution: .001dpcm) { // Target only Safari browsers
+        @supports (-webkit-appearance:none) {         // Use `focus` instead of `focus-visible`
+          &:focus { @include focus-ring; }            // due to Safari's lack of support
+        }
+      }
+      &:focus-visible { @include focus-ring; }
+    }
+    ```
+  </Tab>
+</Tabs>
+
+### Custom Mixins
+
+The process of overriding those mixins is really similar to the custom breakpoints steps:
+
+1. Go to the `/src` folder and create the `styles` folder and inside it. You can run the following command to create them:
+
+<Tabs items={['macOS and Linux', 'Windows']}>
+  <Tab>
+    ```bash copy
+    mkdir src/styles
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy
+    mkdir src\styles
+    ```
+  </Tab>
+</Tabs>
+
+2. Inside the `styles` folder, create a new file called `custom-mixins.scss`
+
+<Tabs items={['macOS and Linux', 'Windows']}>
+  <Tab>
+    ```bash copy
+    touch src/styles/custom-mixins.scss
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy
+    touch src\styles\custom-mixins.scss
+    ```
+  </Tab>
+</Tabs>
+
+3. Choose the mixin you want to override and add it to the file:
+
+```diff copy
```

Comment:
I think this is a little bit confusing. 
How about first, we show how to add the mixin in this file, and then the "override changes applied"?